app.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express')
+
+const { getTopics } = require("./controllers/topics.controllers")
+
+const app = express();
+
+app.use(express.json())
```

Comment:
Ideally we would not include the express.json line until you have a body that you need to use in a endpoint.